src/components/cladeFilterDrawer/clusteringMethodSelect.tsx

```diff
@@ -137,65 +137,67 @@ function ClusteringOptions(props: ClusteringOptionsProps) {
           </Select>
         </FormControl>
       </p>
-      <p>
-        <FormControl>
-          <FormLabel>Then, select clustering algorithm</FormLabel>
-          <RadioGroup
-            aria-labelledby="select clustering method"
-            name="radio-buttons-group"
-            onChange={(e, v: string) =>
-              //@ts-ignore
-              handleClusterMethod(v)
-            }
-            value={state.clusteringMethod}
-          >
-            <FormControlLabel
-              value="none"
-              control={<Radio size="small" />}
-              label="None"
-            />
-            <FormControlLabel
-              value="nextstrain"
-              control={
-                <Radio
-                  disabled={
-                    !checkNextstrainValidity(state.clusteringMetadataField)
-                  }
-                  size="small"
-                />
+      {state.clusteringMetadataField && (
```

Comment:
I don't think I fully understand why this would only render when `state.clusteringMetadataField` exists. Above, the "Select metadata field to cluster on" label always exists, so I think it would be weird to just have emptiness underneath it. Feels like either the preceding label should also disappear when the form choices disappear, or just some of the choices should be disabled until the checking shows they're okay to enable (eg, via `checkNextstrainValidity(state.clusteringMetadataField)`).

Reply:
This was a suggestion from @Janeece 's design review to do progressive disclosure. I believe the reasoning was that it was weird to have a fully disabled form widget visible.